code/processes/rdb.q

```diff
@@ -30,7 +30,9 @@ reloadenabled:@[value;`reloadenabled;0b];                   //if true, the RDB w
                                                             //will clear it's data using reload function (called by the WDB)
 parvaluesrc:@[value;`parvaluesrc;`log];                     //where to source the rdb partition value, can be log (from tp log file name), 
                                                             //tab (from the the first value in the time column of the table that is subscribed for) 
-                                                            //anything else will return a null date which is will be filled by pardefault									
+                                                            //anything else will return a null date which is will be filled by pardefault								
+subfiltered:0b      					    //allows subscription filters to be loaded and applied in the rdb
```

Comment:
can you change this to be the same way that we do the other definitions, i.e. it only gets set to this if not already set
subfiltered:@[value;`subfiltered;0b]
Also can you make sure that the comments align on the RHS - use spaces, not tabs.

Reply:
Changed it now